### PR TITLE
Created repo checks using Github actions. 🤞

### PR DESCRIPTION
**Pull request by zagganas**
Had to upgrade coveralls to version 2.2.0

### CODE REVIEW

File: .github/workflows/checks.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    branches: [ dev ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.allow-error }}
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: ["3.6", "3.7"]
```
Comment (lvarin) [inline .github/workflows/checks.yaml]: I think you should add 3.8, 3.9 and 3.10 to the list, and remove 3.6 (so support in `numpy` anymore).

Comment (zagganas) [reply .github/workflows/checks.yaml]: The PR already has 3.8. I tried to tried to respect the allow the "allow-failures" for 3.8 as specified in travis.yaml. However, I think we should not allow continue-on error and put all versions inside the matrix. What do you think?

Comment (lvarin) [reply .github/workflows/checks.yaml]: I agree. Indeed v3.8 is failing due to an old pytest version. In the PR https://github.com/elixir-cloud-aai/TEStribute/pull/86 pytest is being upgraded to fix the problem

Comment (zagganas) [reply .github/workflows/checks.yaml]: Perfect. I'll fix it and do a new commit.

**Review by lvarin — APPROVED** [review]
LGTM

**Pull request merged**